amr-wind/utilities/sampling/SamplingContainer.cpp

```diff
@@ -95,6 +95,8 @@ void SamplingContainer::initialize_particles(
 {
     BL_PROFILE("amr-wind::SamplingContainer::initialize");
 
+    amrex::Print() << "Start Initialize Particles" << std::endl;
```

Comment:
There are a lot of new print statements added in this PR. Are all of these desired for the final version? Some of them seem more targeted for development and debugging. If they are intended to go into main, I recommend that they be turned on via some verbosity level.